cmd/gen-manifests/main.go

```diff
@@ -208,6 +208,14 @@ func makeManifestJob(name string, imgType distro.ImageType, cr composeRequest, d
 
 type DistroArchRepoMap map[string]map[string][]repository
 
+func (darm DistroArchRepoMap) ListDistros() []string {
+	distros := make([]string, 0, len(darm))
+	for d := range darm {
+		distros = append(distros, d)
+	}
+	return distros
```

Comment:
Sorting these would make it more consistent :)

Reply:
I'll do it, but I don't see why it would matter in this case.

Reply:
Oh ha. I now notice this in in main.go not in distrofactory, sorry about that :)